src/components/recordedit/form-container.tsx

```diff
@@ -52,9 +51,42 @@ const FormContainer = (): JSX.Element => {
 
     return () => {
       sensors?.forEach((sensor) => sensor.detach());
-    }
+    };
   }, []);
 
+  // This useffect is added to set a max-width to select-all-row as the width of the visible area
+  useEffect(() => {
+    const mainResizeSensor = new ResizeSensor(
+      formContainer.current as Element,
+      () => {
+        handleScroll();
+      }
+    );
+
+    return () => {
+      mainResizeSensor.detach();
+    };
+  }, []);
+
+  /* Callback event for scroll functionality on recordedit-form to set a max-width to select-all-row as the
+  width of the visible area */
+  const handleScroll = () => {
```

Comment:
Why do you need to call this `onScroll` of the form? Shouldn't the resizeSensor be enough? If that's the case, you should just move the implementation of this function to be inside the resizeSensor.

Reply:
we have to set max-width when we scroll horizontally across the container.
Its a common function getting called on, onScroll and resizeSensor

Reply:
Ok so let's change its name to be what it actually does instead of just `handleScroll`.